mifosng-android/src/main/res/layout/add_payment_detail.xml

```diff
@@ -67,15 +67,16 @@
                 android:layout_height="wrap_content"
                 android:orientation="horizontal">
                 <TextView
-                    android:layout_width="120dp"
+                    android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_marginTop="16dp"
                     android:textStyle="bold"
                     android:layout_marginLeft="16dp"
+                    android:layout_marginBottom="16dp"
                     android:text="@string/total_charges"/>
 
                 <TextView
```

Comment:
If the amount has a lot of trailing zeroes then it will look bad, please handle that edge case also. Rest everything is fine.

Reply:
Sure I will edit it ASAP